docs/HowTo/Get-Started/Snapshot-Start.md

```diff
@@ -0,0 +1,34 @@
+---
+description: How to start Teku using a recent finalized checkpoint state
+---
+
+# Start Teku from a recent state
+
+You can start Teku from a recent finalized checkpoint state to avoid syncing a node from a
+historic state.
+
+When starting from a recent checkpoint, Teku downloads historic chain data in the background.
+
+!!! important
+
+    You need access to a beacon chain client with [REST API enabled] to download the finalized
+    checkpoint state file.
+
+The following example downloads a finalized checkpoint state file (`state.ssz`) from the beacon
+chain client, and starts Teku with the downloaded file:
+
+1. Download a recent finalized checkpoint state using the
+    [`/eth/v1/debug/beacon/states/`](https://consensys.github.io/teku/#operation/getEthV1DebugBeaconStatesWithState_id) endpoint:
+
+    ```bash
+    curl -o state.ssz -H 'Accept: application/octet-stream' http://localhost:5051/eth/v1/debug/beacon/states/finalized
```

Comment:
With https://github.com/ConsenSys/teku/pull/3762 you can also just pass that URL to the `--initial-state` option and Teku will download it automatically.  This is particularly useful when needing to dynamically spin up Teku nodes as you can point new nodes at any existing node and know it will start with the most recent finalized state.

ie `--initial-state http://other-node:5051/eth/v1/debug/beacon/states/finalized`

Without that PR you can still use a URL but it doesn't work with the standard API (only our non-standard version).  Since this documentation will get released with the next Teku release which should have that PR in we can stick to the standard API and it will all work by then.

Reply:
Thanks. Implemented this feedback.